client/src/components/CaptureMatching/Captures.js

```diff
@@ -0,0 +1,66 @@
+import React, {useState, useEffect} from 'react'
```

Comment:
I'm not sure this component is doing anything other than wrapping `CaptureImage`.